README.md

```diff
@@ -25,28 +25,21 @@ zero-knowledge domain specific language (zkDSL), Kakarot is provable by design.
 This allows for proving the execution of EVM transactions, and makes Kakarot a
 _de facto_ so-called _zkEVM_.
 
-While some zkEVM implementations (see for example
-[Scroll](https://github.com/scroll-tech/zkevm-circuits) or
-[Taiko](https://github.com/taikoxyz/taiko-geth)) tries to prove existing EVM
-implementations (mainly Geth), Kakarot is like another new Geth, but provable by
-design, simply because it runs on the CairoVM.
-
 We strongly believe the CairoVM will provide the best zero-knowledge toolbox in
 the coming years and that the Ethereum network effect will remain prevalent. We
 present to developers an abstraction layer they're familiar with: the EVM. Build
 and deploy as if you were working on Ethereum, be forward compatible with the
 future of zero-knowledge.
 
-Kakarot is a work in progress, and it is not ready for production.
-
 [Kakarot presentations and talks around the world](https://www.youtube.com/playlist?list=PLF3T1714MyKDwjjA8oHizXAdLNx62ka5U)
 
 [Getting started](#getting-started) • [Build](#build) • [Test](#test) •
 [Report a bug](https://github.com/kkrt-labs/kakarot/issues/new?assignees=&labels=bug&template=01_BUG_REPORT.md&title=bug%3A+)
 
 ## Supported opcodes
 
-We support 100% of EVM opcodes and 6 out of 9 precompiles.
+We support 100% of EVM opcodes and
```

Comment:
not yet: should this be synced with releases on sepolia|mainnet?

Reply:
I think that README on main branch should be sync with code on main branch, nothing more or lesss